src/tribol/geom/GeomUtilities.cpp

```diff
@@ -330,9 +330,9 @@ void GlobalTo2DLocalCoords( const real* const RESTRICT pX,
                             real* const RESTRICT pLY, int size )
 {
 
-   if (pLX == nullptr || pLY == nullptr)
+   if (size > 0 && (pLX == nullptr || pLY == nullptr))
```

Comment:
Minor (clarity): a few of these can be `SLIC_ERROR_IF` since there's no other code inside the curly braces

Reply:
Yes, good point. Fixed in this file.